src/build/esbuild.ts

```diff
@@ -163,7 +163,7 @@ export class EsbuildSnapshot implements BuildSnapshot {
   }
 
   get paths(): string[] {
-    return Object.keys(this.#files);
+    return Array.from(this.#files.keys());
```

Comment:
What is this change about?

Reply:
Calling `Object.keys()` on a `Map` gives you the keys of the `Map` object itself and not the keys of the data it holds. I'm not sure how this bug has survived this long

Reply:
huh, nice catch!